mix.lock

```diff
@@ -1,14 +1,17 @@
-%{"certifi": {:hex, :certifi, "0.7.0", "861a57f3808f7eb0c2d1802afeaae0fa5de813b0df0979153cbafcd853ababaf", [:rebar3], []},
```

Comment:
If you run `mix do deps.unlock --unused, deps.clean --unused` it should clean this file up a lot. I don't know if it matters having unused deps in the mix.lock file or not though.